quic/s2n-quic-core/src/path/mod.rs

```diff
@@ -323,9 +311,54 @@ pub fn remote_port_blocked(port: u16) -> bool {
     false
 }
 
+// The below ports are also vulnerable to reflection attacks, but are within
+// the original ephemeral port range of 1024–65535, so there is a chance
+// clients may be randomly assigned them. To address this, we can throttle
+// connections using these ports instead of fully blocking them.
+
+/// List of ports to throttle connections from. This list must be sorted.
+///
+/// Based on https://quiche.googlesource.com/quiche/+/bac04054bccb2a249d4705ecc94a646404d41c1b/quiche/quic/core/quic_dispatcher.cc#498
+const THROTTLED_PORTS: [u16; 3] = [
+    1900,  // SSDP, vulnerable to reflection attacks.
+    5353,  // mDNS, vulnerable to reflection attacks.
+    11211, // memcache, vulnerable to reflection attacks.
+];
+const MAX_THROTTLED_PORT: u16 = THROTTLED_PORTS[THROTTLED_PORTS.len() - 1];
+pub const THROTTLED_PORTS_LEN: usize = THROTTLED_PORTS.len();
```

Comment:
@camshaft do you think it would be cleaner to provide the len in a `const fn` instead? 

Also I am curious if you had an opinion on how these arrays are used to have a `BasicRateLimiter` per port. Usually I would use something from the standard lib 😅

Reply:
The `const` is fine. Limiter per port with an array works great, IMO. No allocations needed, too.